src/test/java/io/xstefank/wildfly/bot/PRDescriptionCheckTest.java

```diff
@@ -75,7 +89,12 @@ void correctLinkCheckSuccessTest() throws IOException {
     @Test
     void multipleLineDescription() throws IOException {
         GitHubAppTesting.given()
-                .github(mocks -> mocks.configFileFromString("wildfly-bot.yml", wildflyConfigFile))
+                .github(mocks -> {
+                    mocks.configFileFromString("wildfly-bot.yml", wildflyConfigFile);
+
+                    PagedSearchIterable<GHPullRequestFileDetail> fileDetails = GitHubAppMockito.mockPagedIterable(mockEmptyFileDetails());
+                    Mockito.when(mocks.pullRequest(1352150111).listFiles()).thenReturn(fileDetails);
```

Comment:
why do you need this? do we pass json that includes changes to the config file?

Reply:
As otherwise all the tests would fail on following line in `ConfigFileChangeProcessor` as `listFiles` would returns by default null.
```suggestion
                    for (GHPullRequestFileDetail changedFile : pullRequest.listFiles()) {
```